concent_api/core/validation.py

```diff
@@ -14,6 +14,7 @@
 from core.constants                 import MESSAGE_TASK_ID_MAX_LENGTH
 from core.constants import VALID_ID_REGEX
 from core.constants import VALID_SHA1_HASH_REGEX
+from core.exceptions import HashingAlgorithmError
```

Comment:
since `HashingAlgorithmError` is defined in `core`, and `HashingAlgorithm` is used in `validation` module only, its defition should be moved from `gatekeeper` to `core`

Reply:
Done